packages/fuels-code-gen/src/program_bindings/abigen/bindings/contract.rs

```diff
@@ -62,8 +64,25 @@ pub(crate) fn contract_bindings(
                 self.account.clone()
             }
 
-            pub fn with_account<U: ::fuels::accounts::Account>(&self, account: U) -> ::fuels::types::errors::Result<#name<U>> {
-                ::core::result::Result::Ok(#name { contract_id: self.contract_id.clone(), account, log_decoder: self.log_decoder.clone()})
+            pub fn with_account<U: ::fuels::accounts::Account>(&self, account: U)
+            -> ::fuels::types::errors::Result<#name<U>> {
+                ::core::result::Result::Ok(
+                #name {
+                        contract_id: self.contract_id.clone(),
+                        account,
+                        log_decoder: self.log_decoder.clone(),
+                        encoder_config: self.encoder_config.clone()
+                })
+            }
```

Comment:
`script_instances` takes ownership and I suggest we do the same here.

```suggestion
            pub fn with_account<U: ::fuels::accounts::Account>(self, account: U)
            -> ::fuels::types::errors::Result<#name<U>> {
                ::core::result::Result::Ok(
                #name {
                        contract_id: self.contract_id,
                        account,
                        log_decoder: self.log_decoder,
                        encoder_config: self.encoder_config
                })
            }
```